arch/arm/src/stm32/hardware/stm32_dma_v1.h

```diff
@@ -64,6 +64,9 @@
 #define DMA_CHAN5                  (4)
 #define DMA_CHAN6                  (5)
 #define DMA_CHAN7                  (6)
+#if defined(CONFIG_STM32_STM32G47XX)
```

Comment:
It's better to introduce a new definition in this file like this 

#if defined(CONFIG_STM32_STM32G47XX)
#define HAVE_DMA_CHAN8 
#endif

We should avoid chip-specific #ifdef's if possible.

Reply:
This is done in commit 710da07dea7a8ac40c00e1093cb697e63d05fb2a. Please see if that is satisfactory.